### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by khoanv99**
Update the game over message so people know how to play again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
   In your repository settings: enable template repository, add your 1280×640 social image, auto delete head branches.
   Add your open source license, GitHub uses Creative Commons Attribution 4.0 International.
 -->
-dsds
+Update the game over message
```
Comment (khoanv99) [inline README.md:7]: comment in here

**Review by khoanv99 — COMMENTED** [review]
comment

File: README.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
   In your repository settings: enable template repository, add your 1280×640 social image, auto delete head branches.
   Add your open source license, GitHub uses Creative Commons Attribution 4.0 International.
 -->
-dsds
```
Comment (rickboi9z) [inline README.md:7]: vfgf

File: README.md (hunk 1 above)
Comment (khoanv99) [reply README.md:7]:
```suggestion
Update the game over message
```

**Review by khoanv99 — COMMENTED** [review]
add suggested

**Pull request merged**